Helper/ScopeSetting.php

```diff
@@ -121,6 +124,11 @@ public function getVersion()
             ->getOne(self::MODULE_NAME)['setup_version'];
     }
 
+    public function getWebhookSecret()
```

Comment:
how come there is no setWebhookSecret ?

Reply:
we don't ever need to set this value, its done through the system.xml file, we just need to retrieve it.

Reply:
Could you explain more? When I look at the below its not clear to me that I see any value

```
                <field id="webhook_secret" translate="label" type="password" sortOrder="10" showInDefault="1" showInWebsite="1" showInStore="1">
                    <label>Webhook Secret</label>
                    <backend_model>Magento\Config\Model\Config\Backend\Encrypted</backend_model>
                </field>
```

Reply:
it's done natively through magento, we don't have to do a set function for this.  we just need to be able to retrieve it.